pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -321,6 +321,39 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 				Providers: Providers{},
 			},
 		},
+		{
+			name: "Config builder should handle quick starts options",
```

Comment:
@sahil143 this works fine, just one idea: there is another test section below which verifies the input against some YAML output. If you like you can add another test there because the final YAML is what the console bridge could parse later. So this verifies the contract between both projects a little better.